### PR TITLE
JBPM-5233: Data source management

### PR DESCRIPTION
**Pull request by wmedvede**
- changes the deployment id separator to avoid WF conflicts in windows systems

### CODE REVIEW

**Comment by wmedvede** [comment]
Hello @pefernan would you mind review this PR?

Thanks!

**Comment by pefernan** [comment]
+1 (when green)

File: kie-wb-common-screens/kie-wb-common-datasource-mgmt/kie-wb-common-datasource-mgmt-backend/src/test/java/org/kie/workbench/common/screens/datasource/management/backend/core/DataSourceManagementTestConstants.java [hunk 1]
```diff
@@ -32,7 +32,7 @@
 
     String DS1_JNID = "jndi1";
 
-    String DS1_DEPLOYMENT_ID = "kie:" + DS1_UUID + ":_" + RANDOM_UUID;
+    String DS1_DEPLOYMENT_ID = "kie#" + DS1_UUID + "#_" + RANDOM_UUID;
```
Comment (psiroky) [inline kie-wb-common-screens/kie-wb-common-datasource-mgmt/kie-wb-common-datasource-mgmt-backend/src/test/java/org/kie/workbench/common/screens/datasource/management/backend/core/DataSourceManagementTestConstants.java]: Would it make sense to have the separator as a constant as well?

**Comment by wmedvede** [comment]
@pefernan  @manstis 
Suggestion added, ready to merge from my side.

Thanks

**Comment by wmedvede** [comment]
Hello @csadilek , I don't no why this PR is still in the waiting queue, but it's ready to merge.

**Comment by manstis** [comment]
@wmedvede probably because those that are able to merge PRs do not constantly scan open PRs across all repositories waiting for either comments to be resolved or builds to go green. IMO it's up to the person who raised a PR(s) to follow the submission through review to merge and closure. If that person needs to request reviews or merge (when reviewed and green) they that is their responsibility.

**Pull request merged**